<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HisCurve</class>
 <widget class="QDialog" name="HisCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QChartView" name="hisCurve_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QChartView" name="hisCurve_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QChartView" name="hisCurve_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QChartView" name="hisCurve_6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QChartView" name="hisCurve_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QChartView" name="hisCurve_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QChartView" name="hisCurve_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QChartView" name="hisCurve_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>快速查询：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>开始时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>结束时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>数据导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>187</width>
       <height>507</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
background:rgba(255,255,255,1);
border:1px solid rgba(191,191,191,1);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_1">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_blue.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_2">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_green.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_3">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_chart_3">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_pink.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_4">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_red.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_5">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_yellow.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_6">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_indigo.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_7">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_darkerblue.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_chart_8">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>TT-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="minimumSize">
           <size>
            <width>42</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
image: url(:/curve_purple.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-size:14px;
font-family:PingFang SC;
font-weight:300;
line-height:44px;
color:rgba(97,97,97,1);
}</string>
          </property>
          <property name="text">
           <string>50°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="TT01_TT08_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>TT01-TT08</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TT09_TT16_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>TT09-TT16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TT17_TT24_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>TT17-TT24</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TT25_TT32_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TT25-TT32</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TT33_TT36_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>TT33-TT36</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pressure_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>压力</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flow_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>流量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="speed_1_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>调速1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="speed_2_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>调速2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="others_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>其他</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
